rasa/core/policies/ensemble.py

```diff
@@ -694,43 +629,50 @@ def _best_policy_prediction(
 
         return self._pick_best_policy(predictions)
 
-    def _fallback_after_listen(
-        self, domain: Domain, prediction: PolicyPrediction
+    @staticmethod
+    def _get_prediction(
```

Comment:
What is this new method for?

Reply:
This was accidentally added when I was resolving merge conflicts with main. I have removed it.